### DISCUSSION TITLE
spherical camera

### DISCUSSION THREAD

**Discussion by Gzhji** [mitsuba-renderer/mitsuba3#536] [Feature requests] [answered]
Dear Community,

MItsuba 0.6 can set: sensor type="spherical", but seems it is not available in Mitsuba3
https://mitsuba.readthedocs.io/en/stable/src/generated/plugins_sensors.html

Does anyone know how to set the spherical camera in MItsuba3?

Thanks!

**Comment by dvicini** [comment, marked as answer]
Hi Gzhji, 

Currently the spherical sensor is indeed not implemented in Mitsuba 3. I once implemented a spherical sensor as a Python plugin. Feel free to use this one or use it as an inspiration how to forward port the implementation from Mitsuba 0.6:
https://github.com/rgl-epfl/differentiable-sdf-rendering/blob/main/figures/spherical_sensor.py

Best,
Delio
